modules/cert-manager-nw-policy-params.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-nw-policy.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="cert-manager-nw-policy-params_{context}"]
+= Network policy configuration parameters
+
+You can enable and configure network policies for the {cert-manager-operator} by updating the CertManager custom resource (CR). The CR includes the following parameters for enabling default network policies and defining custom egress rules.
+
+[cols="1,3", options="header"]
+|===
+| Field
+| Type
+| Description
+
+|`spec.defaultNetworkPolicy`
+|`boolean`
+|Specifies whether to enable the default network policy for the cert-manager components.
```

Comment:
```suggestion
|Specifies whether to enable the default network policy for the cert-manager components. Default network policies cannot be disabled once enabled to prevent accidental security degradation. Users should carefully plan their network policy requirements before enabling this field.
```